scanner/src/main/kotlin/storages/ClearlyDefinedStorage.kt

```diff
@@ -127,7 +127,7 @@ class ClearlyDefinedStorage(
         try {
             readFromClearlyDefined(id, packageCoordinates(id, vcs, sourceArtifact))
         } catch (e: IllegalArgumentException) {
-            log.warn("Could not obtain ClearlyDefined coordinates for package '${id.toCoordinates()}'.", e)
+            log.warn { "Could not obtain ClearlyDefined coordinates for package '${id.toCoordinates()}'." }
```

Comment:
You should also add `e.showStackTrace()`, especially as you mention `--stacktrace` in the commit message. Or you could add ": $e.collectMessagesAsString()" as we do in other places. Or both.

Reply:
I decided to only add the call to `e.showStackTrace()` as seeing the exception message as part of the log was exactly what [confused a user](https://github.com/oss-review-toolkit/ort/issues/3239).

Reply:
We use this order and formatting everywhere else in the code:
```
...catch(e: Exception) {
  e.showStackTrace()

  log.warn { ... }

  ...
}
```